namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/main.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  backend "s3" {}
+}
+
```

Comment:
Please reinstate the default AWS provider here. Otherwise the pipeline will fail for modules which don't use the 'aws.london' provider.